.golangci.yml

```diff
@@ -56,207 +80,487 @@ output:
 # all available settings of specific linters
 linters-settings:
   staticcheck:
-    # Select the Go version to target. The default is '1.13'.
-    go: "1.19"
-    # https://staticcheck.io/docs/options#checks
+    # Select the Go version to target.
+    # Default: "1.13"
+    # Deprecated: use the global `run.go` instead.
+    # go: "1.15"
+    # SAxxxx checks in https://staticcheck.io/docs/configuration/options/#checks
+    # Default: ["*"]
```

Comment:
Is it really still 1.13?

Reply:
This file was generated from the default config file for .golangci-lint; so I assume this is correct